MVP/MVP/MVP.iOS/AppDelegate.cs

```diff
@@ -24,29 +24,37 @@ public partial class AppDelegate : global::Xamarin.Forms.Platform.iOS.FormsAppli
         //
         public override bool FinishedLaunching(UIApplication app, NSDictionary options)
         {
-			var tint = UIColor.FromRGB(0,79,138);
-			// Some basic navigationbar styling.
-			UINavigationBar.Appearance.BarTintColor = UIColor.FromRGB(250, 250, 250); //bar background
-			UINavigationBar.Appearance.TintColor = tint; //Tint color of button items
+            var tint = UIColor.FromRGB(0, 79, 138);
+            // Some basic navigationbar styling.
+            UINavigationBar.Appearance.BarTintColor = UIColor.FromRGB(250, 250, 250); //bar background
+            UINavigationBar.Appearance.TintColor = tint; //Tint color of button items
 
-			UIBarButtonItem.Appearance.TintColor = tint; //Tint color of button items
+            UIBarButtonItem.Appearance.TintColor = tint; //Tint color of button items
 
-			UITabBar.Appearance.TintColor = tint;
+            UITabBar.Appearance.TintColor = tint;
 
-			UISwitch.Appearance.OnTintColor = tint;
+            UISwitch.Appearance.OnTintColor = tint;
 
-			UIAlertView.Appearance.TintColor = tint;
+            UIAlertView.Appearance.TintColor = tint;
 
-			UIView.AppearanceWhenContainedIn(typeof(UIAlertController)).TintColor = tint;
-			UIView.AppearanceWhenContainedIn(typeof(UIActivityViewController)).TintColor = tint;
-			UIView.AppearanceWhenContainedIn(typeof(SLComposeViewController)).TintColor = tint;
+            UIView.AppearanceWhenContainedIn(typeof(UIAlertController)).TintColor = tint;
+            UIView.AppearanceWhenContainedIn(typeof(UIActivityViewController)).TintColor = tint;
+            UIView.AppearanceWhenContainedIn(typeof(SLComposeViewController)).TintColor = tint;
 
-			global::Xamarin.Forms.Forms.Init();
+            global::Xamarin.Forms.Forms.Init();
+
+            ImageCircle.Forms.Plugin.iOS.ImageCircleRenderer.Init();
 
-			ImageCircle.Forms.Plugin.iOS.ImageCircleRenderer.Init();
-            //Xamarin.FormsMaps.Init();
             LoadApplication(new App());
 
+            foreach (NSString family in UIFont.FamilyNames)
```

Comment:
Can we remove this?

Reply:
I removed it, had definetly served its use. Also removed some statusbar code that became obsolete due to the fix in navbar/statusbar colors.